scripts/__utils__.sh

```diff
@@ -5,8 +5,14 @@ if [ "${__utils__}" = true ] ; then
 fi
 
 function sha256Hash() {
-  local output=$(printf "$1" | openssl dgst -sha256)
-  printf "${output}"
+  uname | grep -q ^Darwin
+  if [ $? -eq 0 ]; then
+    local output="$(printf "$1" | ${OPENSSL_BIN} dgst -sha256 -binary | xxd -p -c 256)"
+  else
+    local output="$(printf "$1" | ${OPENSSL_BIN} dgst -sha256)"
+  fi
+
+  echo "${output}"
```

Comment:
Could we use "printf"? since "echo" return new-line-char which might made some unexpected errors in some env